### PR TITLE
Syntax Editor Macro to populate macrosObj with records from the Syntax Editor Macro table

### PR DESCRIPTION
**Pull request by snamjosh**

### CODE REVIEW

**Review by SapphicFire — COMMENTED** [review]
This submission is not relevant to this repository. We encourage contributors to review the project's objectives and guidelines (see the README.md and CONTRIBUTING.md files) before submitting pull requests. 

This pull request has been copied from a community answer without proper attribution. Contributions should be original, and any referenced work should be properly attributed.

Source community answer with code copied and very minor alterations: [Service Portal: Syntax Editor Macros](https://www.servicenow.com/community/now-platform-articles/service-portal-syntax-editor-macros/ta-p/2311306). This is linked on the README.md file on the core repo

I am closing this and am opting to mark this as spam as I feel this was not done in good faith. As a fellow maintainer I am deeply disappointed.

**Comment by snamjosh** [comment]
@SapphicFire first of all, this is not a copied code since I am using it from years when Macros were introduced on the platform.how do you know that article you posted didn’t copied from me? I have been around this platform since 2013 and I have provided lot of solutions to hundreds of developers.

secondly, you need to brush your technical skills since I have this macro running  on my instance.

i am highly dissatisfied in your technical skills.

File: Service Portal/spOnReady/readme.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This syntax macro Get data from the background and make it accessible after it is retrieved.
+
+e.g This macro can be used during server script during service portal widget development.
```
[the remark this answers is not in the document]
Comment (snamjosh) [reply Service Portal/spOnReady/readme.js:3]: I agree with this comment.

[the remark this answers is not in the document]
Comment (Lacah) [reply Service Portal/spOnReady/readme.js:3]: Can you plesase update the radme so this review can be closed and the PR merged? Thanks.

**Review by Lacah — CHANGES_REQUESTED** [review]

File: Service Portal/spOnReady/readme.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (snamjosh) [reply Service Portal/spOnReady/readme.js]: @Lacah i updated readme to include review comments.

**Pull request closed without merging**